engine/src/main/java/com/datatorrent/stram/debug/StdOutErrLog.java

```diff
@@ -54,6 +56,7 @@ public static void tieSystemOutAndErrToLog()
 
     System.setOut(createLoggingProxy(System.out));
     System.setErr(createLoggingProxy(System.err));
+    LoggerUtil.addAppender("apex");
```

Comment:
The dynamic appender itself can generate some log events during its initialization if there is an incorrectness in the appender parameters or environment. So I think it is better to add the dynamic appender after the reconfiguration of the regular DT appender. At least, the user can find the corresponder log event records in the regular log, if the dynamic appender was not started.

But I think I can move the adding of the dynamic appender before createLoggingProxy calls in the method tieSystemOutAndErrToLog().